assessments/0011-keda/keda-analysis.md

```diff
@@ -0,0 +1,416 @@
+
+### New user content
+
+Clearly label the starting point for new users. Make sure that there is an easy to follow workflow for installation, configuring and running a basic scaler, and any other tasks required to begin using KEDA.
+
+### Content maintainability & site mechanics
+
+No recommendations
+
+### Content creation processes
+
+Make it easier to find the instructions for releasing a new version of the documentation and updating the documentation. 
+
+Document in the repo who the website/documentation maintainers are. 
+
+### Inclusive language
+
+Remove non-inclusive language throughout the documentation as recommended on the [inclusive naming website](https://inclusivenaming.org/).
+
+
+# Contributor documentation
+
+KEDA is a **graduated** project of CNCF. This means that the project should have [*very high*](https://github.com/cncf/techdocs/blob/main/assessments/criteria.md) standards for contributor documentation.
+
+| Criterion                                 | Rating (1-5)           |
+| ---                                       | ---                    | 
+| Communication methods documented          |  3 - meets standards   | 
+| Beginner friendly issue backlog           |  3 - meets standards   |
+| “New contributor” getting started content |  3 - meets standards   | 
+| Project governance documentation          |  4 - meets or exceeds standards   | 
+
+
+## Comments
+
+### Communication methods documented
+
+The KEDA main repo points to **community** resources, including a KEDA Slack channel in the Kubernetes workspace. There is a **community page** on the website that invites readers to a **biweekly Zoom meeting**. The main website has direct links in the footer to the Slack community, the **GitHub** repo, and a Twitter feed. I could not find any mention of a **mailing list**. 
+
+
+### Beginner friendly issue backlog
+
+Doc **issues** in the repo are **well documented** and have been labeled and, presumably, **triaged**. 
+
+There is a **good first issue** label (although this label is not currently applied to any issues).
+
+There are **stale issues**, but they seem for the most part to be managed. There are only ~30 open issues in the doc repo.
+
+
+### New contributor getting started content
+
+The website contains a **community section**. 
+
+There is **CONTRIBUTORS** document in the website/documentation repo with instructions for **getting help** and building and contributing new documentation. 
+
+
+### Project governance documentation
+
+**Governance** is clearly documented in its own repo. Presumably this information is applicable to all of the repos in kedacore.
+
+
+## Recommendations
+
+### Communication methods documented
+
+If there is a mailing list or other news distribution channel, add it to the community page. 
```

Comment:
This is not the case. What is the recommendation then? Should we explicitly call this out on the community page? This might feel a bit odd then, no?

Reply:
I don't think lack of a newsletter is a problem. All I'm saying here is that if it exists I couldn't find it. The project seems to have very healthy communication channels, so I wouldn't worry about it.